sign/rpmgensig.c

```diff
@@ -461,15 +461,56 @@ static rpmRC includeVeritySignatures(FD_t fd, Header *sigp, Header *hdrp)
     rpmRC rc = RPMRC_OK;
     char *key = rpmExpand("%{?_file_signing_key}", NULL);
     char *keypass = rpmExpand("%{?_file_signing_key_password}", NULL);
+    char *pkcs11_engine = rpmExpand("%{?_pkcs11_engine}", NULL);
+    char *pkcs11_module = rpmExpand("%{?_pkcs11_module}", NULL);
+    char *pkcs11_keyid = rpmExpand("%{?_pkcs11_keyid}", NULL);
     char *cert = rpmExpand("%{?_file_signing_cert}", NULL);
     char *algorithm = rpmExpand("%{?_verity_algorithm}", NULL);
     uint16_t algo = 0;
 
+    if (rstreq(key, "")) {
+	free(key);
```

Comment:
Be careful to follow the existing indentation style: it's tabs *and* spaces - most editors end up picking one and this looks weird.